manifest.json

```diff
@@ -36,8 +37,10 @@
 	"web_accessible_resources": [
         "images/*",
         "libs/alertify.min.js",
+        "libs/jquery-3.4.1.min.js",
```

Comment:
humm.. I never noticed I had a mix of tabs and spaces here, I'll need to fix that!